project-base/storefront/components/Blocks/Banners/BannersSlider.tsx

```diff
@@ -15,16 +17,16 @@ export const BannersSlider: FC<BannersSliderProps> = ({ sliderItems, dataTestId
     const [pause, setPause] = useState(false);
     const timer = useRef<NodeJS.Timer | null>(null);
     const sliderBoxRef = useRef<HTMLDivElement>(null);
+
+    const { width: windowWidth } = useGetWindowSize();
+    const isDesktop = windowWidth > desktopFirstSizes.tablet;
+    const isRecognizingWindowWidth = windowWidth < 0;
```

Comment:
I would say this name is incorrect, isn't it? If the width is below 0, meaning -1, that's when the window width is not recognized, right? Shouldn't it be named the other way around?

Reply:
That's why it is in present continuous with "ing" which is state exactly for what is needed - state when window width is not recognized yet, but still recognizing. Are we on the same page now?